src/cronjobs/monthlyActivity.tsx

```diff
@@ -0,0 +1,108 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Let's use `src/scripts/cronjobs`

Reply:
With apologies for rebasing, but I was afraid I wasn't going to be able to get all the path changes correct if I spread this over fixup commits. Fixed in f753a0f952a056c5a4ed2cf0c1611ece8e076bb4 and 93c298a1b29e9e135eb1d2d5014a7892d9d62890.